helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -302,6 +300,76 @@ processors:
      - k8s.namespace.name
      - k8s.pod.name
      - k8s.pod.uid
+  {{- else }}
+  resource/splunk:
+    attributes:
+    - key: host.name
+      from_attribute: k8s.node.name
+      action: upsert
+    - key: com.splunk.sourcetype
+      from_attribute: k8s.pod.annotations.splunk.com/sourcetype
+      action: upsert
+    - key: com.splunk.index
+      from_attribute: k8s.namespace.annotations.splunk.com/index
+      action: upsert
+    - key: service.name
+      from_attribute: k8s.pod.name
+      action: upsert
```

Comment:
Why this is needed?

Reply:
I thought this is needed for O11y

Reply:
Everything needed for o11y is already here